rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -4239,8 +4243,10 @@ bool ppu_initialize(const ppu_module& info, bool check_only)
 
 		named_thread_group threads(fmt::format("PPUW.%u.", ++g_fxo->get<thread_index_allocator>().index), thread_count, [&]()
 		{
-			// Set low priority
-			thread_ctrl::scoped_priority low_prio(-1);
+#ifdef _WIN32
+		// Set low priority
+		thread_ctrl::scoped_priority low_prio(-1);
+#endif
```

Comment:
Why are you adding these ifdefs everywhere?
There's no reason to do that.

Reply:
remnants from personal testing. removed.